jinahub/indexers/storage/PostgreSQLStorage/postgres_indexer.py

```diff
@@ -57,6 +60,7 @@ def __init__(
         self.database = database
         self.table = table
         self.logger = get_logger(self)
+        self.total_shards = total_shards
```

Comment:
This is a semantically different total_shards, then in the `FaissPostgresSearcher`. Can we have a better naming?

Reply:
Yes, renamed to virtual shards